org.eclipse.xtext.ui.tests/build.properties

```diff
@@ -13,4 +13,6 @@ bin.includes = model/generated/,\
                icons/,\
                testfiles/
 src.includes = about.html
+additional.bundles = org.eclipse.emf.mwe2.launch,\
```

Comment:
Looks like the handling of the mwe2 dependency is not consistent yet.

Reply:
fixed